commands/build.go

```diff
@@ -26,7 +26,7 @@ type BuildFlags struct {
 	Buildpacks []string
 }
 
-func Build(logger logging.Logger, config *config.Config, packClient *pack.Client) *cobra.Command {
+func Build(logger logging.Logger, cfg config.Config, packClient *pack.Client) *cobra.Command {
```

Comment:
It doesn't seem like `AdditionalMirrors` is being passed in to `Build`. How do local run image mirrors get set?

Reply:
Did our acceptance tests not catch that or are we missing something?